enaml/qt/qt_window.py

```diff
@@ -10,7 +10,8 @@
 from atom.api import Typed, atomref
 
 from enaml.layout.geometry import Pos, Rect, Size
-from enaml.widgets.window import ProxyWindow, CloseEvent
+from enaml.widgets.window import ProxyWindow
+from enaml.widgets.close_event import CloseEvent
```

Comment:
`close_event` should be imported before `window` to maintain sort order. /pedantry